adafruit_esp32spi/adafruit_esp32spi_wifimanager.py

```diff
@@ -144,7 +144,7 @@ def connect_normal(self):
                 self.esp.connect_AP(bytes(ssid, "utf-8"), bytes(password, "utf-8"))
                 failure_count = 0
                 self.pixel_status((0, 100, 0))
-            except (ValueError, RuntimeError) as error:
+            except (ValueError, OSError) as error:
```

Comment:
A `ValueError` here now indicates something that is never going to work, so I think it should no longer be caught here, and instead should just propagate up.
```suggestion
            except OSError as error:
```

Reply:
Done